File: MOB/InterestingCalculation/InterestingCalculation/MainActivity.cs
using Android.App;
using Android.Widget;
using Android.OS;
using System;

namespace InterestingCalculation
{
    [Activity(Label = "InterestingCalculation", MainLauncher = true)]
    public class MainActivity : Activity
    {
        public EditText InitialAmount { get; private set; }
        public EditText InterestRate { get; private set; }
        public EditText MonthAmount { get; private set; }
        public EditText FinalAmount { get; private set; }
        public Button ButtonClear { get; private set; }
        public Button ButtonCalculate { get; private set; }

        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);

            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.Main);
            //
            InitialAmount = FindViewById<EditText>(Resource.Id.editTextInitialAmount);
            InterestRate = FindViewById<EditText>(Resource.Id.editTextInterestRate);
            MonthAmount = FindViewById<EditText>(Resource.Id.editTextMonthAmount);
            FinalAmount = FindViewById<EditText>(Resource.Id.editTextFinalAmount);
            ButtonClear = FindViewById<Button>(Resource.Id.buttonClear);
            ButtonCalculate = FindViewById<Button>(Resource.Id.buttonCalculate);
            //
            ButtonClear.Click += ButtonClear_Click;
            ButtonCalculate.Click += ButtonCalculate_Click;
        }

        private void ButtonClear_Click(object sender, System.EventArgs e)
        {
            InitialAmount.Text = string.Empty;
            InterestRate.Text = string.Empty;
            MonthAmount.Text = string.Empty;
            FinalAmount.Text = string.Empty;
            //
            Toast.MakeText(Application.Context, Resource.String.fields_cleared, ToastLength.Long).Show();
        }

        private void ButtonCalculate_Click(object sender, System.EventArgs e)
        {
            // Values to Calculate
            Double initialAmount = 0, interestRate = 0;
            Int32 monthAmount = 0;
            // Texts from View
            var initialAmountStr = InitialAmount.Text;
            var interestRateStr = InterestRate.Text;
            var monthAmountStr = MonthAmount.Text;
            //
            var errorMessage = String.Empty;
            if(!Double.TryParse(initialAmountStr, out initialAmount))
            {
                if (errorMessage.Length != 0)
                    errorMessage += "\n";
                errorMessage += "Fill the field Initial Amount correctly!";
            }
            if (!Double.TryParse(interestRateStr, out interestRate))
            {
                if (errorMessage.Length != 0)
                    errorMessage += "\n";
                errorMessage += "Fill the field Interest Rate correctly!";
            }
            if (!Int32.TryParse(monthAmountStr, out monthAmount))
            {
                if (errorMessage.Length != 0)
                    errorMessage += "\n";
                errorMessage += "Fill the field Month Amount correctly!";
            }
            //
            if(String.IsNullOrWhiteSpace(errorMessage))
            {
                var finalAmount = initialAmount + (interestRate / 100 * initialAmount * monthAmount);
                FinalAmount.Text = finalAmount.ToString();
            }
            else
            {
                // Something is Wrong
                Toast.MakeText(Application.Context, errorMessage, ToastLength.Long).Show();
            }
        }
    }
}

File: MOB/STDevelopment/app/src/main/java/sp/senai/stdevelopment/activities/MainActivity.java
package sp.senai.stdevelopment.activities;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import sp.senai.stdevelopment.R;

public class MainActivity extends AppCompatActivity {

    private Button btnLogin;
    private EditText etUsername;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        //
        btnLogin = findViewById(R.id.btnLogin);
        etUsername = findViewById(R.id.etUsername);
        //
        btnLogin.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String username = etUsername.getText().toString();
                if(username.length() < 1) {
                    Toast.makeText(MainActivity.this, "You can't make login with an empty name!", Toast.LENGTH_SHORT).show();
                }
                else {
                    Intent intent = new Intent(getBaseContext(), InformativeActivity.class);
                    intent.putExtra("username", username);
                    startActivity(intent);
                }
            }
        });
    }
}
